src/client/caseCorrect.js

```diff
@@ -1,7 +1,8 @@
 var yggdrasil = require('yggdrasil')({});
 var UUID = require('uuid-1345');
 
-module.exports = function(client, options) {
+module.exports = function(client) {
+  var options = client.options;
```

Comment:
ah that kind of thing `https://github.com/PrismarineJS/node-minecraft-protocol/pull/331/files#diff-9053b72ce1984293dcb954a0ccc89097R9`.
Couldn't it be put in a client.wait_connect ?
But the problem of relying on the order of execution of plugins remains.